lib/crypto.js

```diff
@@ -530,6 +530,10 @@ ECDH.prototype.getPublicKey = function getPublicKey(encoding, format) {
   return key;
 };
 
+ECDH.prototype.validKeyPair = function() {
```

Comment:
Bike-shed time: I don't really like the name of this method.  Calling it e.g. `.isValidKeyPair()` more accurately conveys that it returns a boolean.

(Or maybe just `.isValid()`?  Are there other ways a ECDH instance can be invalid?)